alodataset/sintel_base_dataset.py

```diff
@@ -55,7 +55,7 @@ class SintelBaseDataset(BaseDataset, SequenceMixin):
         "temple_3",
     ]
 
-    def __init__(self, cameras=None, labels=None, passes=None, sintel_sequences=None, name="Sintel_base", **kwargs):
+    def __init__(self, training=True,cameras=None, labels=None, passes=None, sintel_sequences=None, name="Sintel_base", **kwargs):
```

Comment:
we support Split dataset in `alodataset`. It's cleaner to  use that for train/val split.

```
from alodataset import BaseDataset, Split, SplitMixin

class SintelBaseDataset(BaseDataset, Split, SequenceMixin):

SPLIT_FOLDERS = {Split.VAL: "validation", Split.TRAIN: "training"}
path = os.path.join(dset_dir, self.split, ...)
```

In that code `self.split` is `Split.VAL` or `Split.TRAIN`

Reply:
Why can't it be `Split.TEST` since that's the point of issue #38 ?

Reply:
it can be `Split.TEST` for test dataset